src/Graphics/src/Graphics/Platforms/iOS/PlatformPdfExportContext.cs

```diff
@@ -28,7 +28,7 @@ protected override void AddPageImpl(float width, float height)
 
 			if (_tempFilePath == null)
 			{
-				_tempFilePath = Path.GetTempFileName();
+				_tempFilePath = Path.GetRandomFileName();
```

Comment:
Can you share a link discussing `GetTempFileName()`? The one I found was:

* https://docs.datadoghq.com/code_analysis/static_analysis_rules/csharp-security/unsafe-temp-file/

I actually don't think this code is correct, as `GetTempFileName()` returns a full path and you have to use `Path.Combine(Path.GetTempPath(), Path.GetRandomFileName())` otherwise.

Did you also check the entire codebase? I find it hard to believe this was used in one place.

Reply:
If we really want to "ban" `GetTempFileName()` we could also list it here:
* https://github.com/dotnet/maui/blob/main/eng/BannedSymbols.txt

And anyone would get a build error if they used it in a non-test project.